src/init/index.ts

```diff
@@ -1,15 +1,70 @@
 import * as Generator from "yeoman-generator";
 import * as chalk from "chalk";
-import parseFabloConfig from "../utils/parseFabloConfig";
 import { GlobalJson } from "../types/FabloConfigJson";
 
+const DEFAULT_FABLO_CONFIG = {
+  "$schema": "https://github.com/hyperledger-labs/fablo/releases/download/2.2.0/schema.json",
+  "global": {
```

Comment:
typically in JS we don't have quotes around object parameters if not needed. Could you remove it? Also please add strict type for `DEFAULT_FABLO_CONFIG`, as it was before:
```suggestion
const DEFAULT_FABLO_CONFIG: FabloConfigJson = {
  "$schema": "https://github.com/hyperledger-labs/fablo/releases/download/2.2.0/schema.json",
  global: {
```

(TS build error fix might be required)